Lib/test/test_minidom.py

```diff
@@ -1631,5 +1631,21 @@ def test_toprettyxml_with_attributes_ordered(self):
                          '<?xml version="1.0" ?>\n'
                          '<curriculum status="public" company="example"/>\n')
 
+    def testCDATAWriting(self):
```

Comment:
First, note that all other methods in this test class use underscores and not CamelCase names. Please adapt the name to the common pattern. Secondly, the tests in this file are very fine granular, which suggests that this should be two tests, not one. Then, the tests should compare the result to a fixed expected value and not allow equal but arbitrary garbage results to succeed. And finally, is it necessary to construct the DOM tree through the API, or could this also use the parser, like the similar tests above?